services/hexathons/src/models/hexathonUser.ts

```diff
@@ -98,6 +109,40 @@ const hexathonUserSchema = new Schema<HexathonUser>({
     ],
     default: [],
   },
+  profile: {
+    matched: {
+      type: Boolean,
+      required: true,
+      default: false,
+    },
+    school: {
+      type: String,
+      enum: commonDefinitions.university.enum,
+    },
+    year: {
+      type: String,
+      enum: commonDefinitions.year.enum,
+    },
+    major: {
+      type: String,
+      enum: commonDefinitions.major.enum,
+    },
+    description: {
+      type: String,
+      maxLength: 200,
+    },
+    commitmentLevel: {
+      type: String,
+      enum: ["High", "Medium", "Low"],
```

Comment:
Let's create an enum `Commitment` similar to how we have [one](https://github.com/HackGT/api/blob/9baf9f7f9641cc419752287cb857bb77dea31306/services/hexathons/src/models/event.ts#LL10C13-L10C22) for `EventType` for instance. And for enum here, just put `Commitment`.